pages/index.vue

```diff
@@ -34,6 +44,16 @@ const spotify = namespace('spotify');
     Welcome,
     CurrentSong,
   },
+  data() {
+    return {
+      customColor: '#000000',
+    };
+  },
+  methods: {
+    updateColorValue(color) {
+      sessionStorage.setItem('color', color);
```

Comment:
Same comment here, we must save this configuration in the query string to be able to apply it in OBS when we copy the URL